CWP/papers/HSF-CWP-2017-14_recotrigger/a2tex.py

```diff
@@ -0,0 +1,231 @@
+#! /usr/bin/env python
```

Comment:
This is a copied version of what @jouvin has for the main document - is it really needed?

Reply:
Probably not indeed -- I was following the principle of minimal cognitive effort when preparing this MR.